nc/source/ui/editor/EditorControls.h

```diff
@@ -247,13 +248,13 @@ namespace nc::ui::editor::controls
     /** @todo this will eventually need to be generic */
     void ComponentSystems(Registry* registry)
     {
-        ComponentSystemHeader<Collider>("Collider", registry->ViewAll<Collider>());
-        ComponentSystemHeader<NetworkDispatcher>("NetworkDispatcher", registry->ViewAll<NetworkDispatcher>());
-        ComponentSystemHeader<ParticleEmitter>("Particle Emitter", registry->ViewAll<ParticleEmitter>());
-        ComponentSystemHeader<PhysicsBody>("Physics Body", registry->ViewAll<PhysicsBody>());
-        ComponentSystemHeader<Transform>("Transform", registry->ViewAll<Transform>());
-        ComponentSystemHeader<nc::MeshRenderer>("Mesh Renderer", registry->ViewAll<MeshRenderer>());
-        ComponentSystemHeader<nc::PointLight>("Point Light", registry->ViewAll<PointLight>());
+        ComponentSystemHeader<Collider>("Collider", view<Collider>(registry));
+        ComponentSystemHeader<NetworkDispatcher>("NetworkDispatcher", view<NetworkDispatcher>(registry));
+        ComponentSystemHeader<ParticleEmitter>("Particle Emitter", view<ParticleEmitter>(registry));
+        ComponentSystemHeader<PhysicsBody>("Physics Body", view<PhysicsBody>(registry));
+        ComponentSystemHeader<Transform>("Transform", view<Transform>(registry));
+        ComponentSystemHeader<nc::MeshRenderer>("Mesh Renderer", view<MeshRenderer>(registry));
```

Comment:
do we still need these nc:: namespaces?

Reply:
Nope, don't event know why they're there.